src/game/InputConstants.h

```diff
@@ -1,4 +1,66 @@
 #pragma once
+#ifndef _INPUTCONSTANTS_H
```

Comment:
The pragma once in the line above acts as a header guard. No need for two layers of header guards.